tests/draft4/items.json

```diff
@@ -19,6 +19,14 @@
                 "description": "ignores non-arrays",
                 "data": {"foo" : "bar"},
                 "valid": true
+            },
+            {
+                "description": "not array is valid",
```

Comment:
That is relevant for Javascript validators (almost half of all validators) that do not differentiate between arrays and pseudo-array (objects that look like arrays). Objects should pass even if array would fail. Quite a few validators fail this test.